### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by deed02392**
add basic dockerfile that drops into a bash shell with cgt-calc on path from a pipx venv

### CODE REVIEW

**Comment by vmartinv** [comment]
What do you use this for? Could you also add a section in the README with some commands?

**Comment by deed02392** [comment]
@vmartinv this Docker container means you don't have to install LaTeX on your host (which can take a while). I made this Dockerfile with all the necessary dependencies, so that if you want, you could officially link this repo with say hub.docker.com and then people could simply `docker pull capital-gains-calculator` and not have to worry about separate build instructions etc..

I updated the README.md with instructions on how it works. The gist is you get the tool running in the container and then just share your tax files with it, run the tool and the output PDF gets dropped in the same directory on your host. You could even generate the report with a single command, rather than get an interactive shell in the container first.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    wget
+
+
+RUN luaotfload-tool --update
+RUN apk --no-cache add py3-pandas
+RUN apk --no-cache add pipx
+RUN pipx install cgt-calc
+RUN pipx ensurepath
+# workaround https://github.com/KapJI/capital-gains-calculator/issues/470
+RUN pipx inject cgt-calc pandas
```
Comment (KapJI) [inline Dockerfile]: This should be no longer needed.

Comment (deed02392) [reply Dockerfile]: rebuilt, tested, confirmed

**Comment by KapJI** [comment]
Please format README with prettier to pass CI.

**Comment by KapJI** [comment]
Thanks!

**Pull request merged**